crt_portal/cts_forms/models.py

```diff
@@ -32,6 +32,11 @@
 User = get_user_model()
 
 
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    intake_filters = models.TextField(max_length=500, blank=True)
+
```

Comment:
@hakhalid11, good opportunity here to provide a `__str__` method so there's a useful string representation of these instances.